asoc_automation_iast/AsocUtils.py

```diff
@@ -191,11 +192,12 @@ def create_app(token, app_name, asset_group, host=ASOC_API, retries=0):
 def get_app_name_by_id(app_id, token, host=ASOC_API):
     url = url_join(host, "/Apps")
     headers = {"Accept": "application/json", "Authorization": "Bearer " + token}
-    params = {"id": app_id}
+    "Change found without running any application"

```

Comment:
there is a test called test_asoc_user_script.py, i think it uses all these interfaces